pegasus/sites.v3/code.org/public/hourofcode/overview.haml

```diff
@@ -0,0 +1,21 @@
+---
+title: Hour of Code Teacher Resources
+nav: hoc_teacher_resources_nav
+theme: responsive
+---
+%link{:href=>"/css/student.css", :rel=>"stylesheet"}
+%link(rel='stylesheet' type='text/css' href='/shared/css/course-blocks.css')
+%script{type: "text/javascript", src: "/js/hoc-overview.js"}
+
+%h1 Hour of Code for all ages
+
+.hoc-tiles-container
+
+.full-resource-block
+  %img.side-image.left.col-50-student{src: "/images/hoc_overview.png"}
+  .side-description-area.right.col-50-student
+    .description
+      %h2.resource-title More Activities from Partners
+      The Hour of Code website has over 100 computer science activities from Code.org and partners for all ages and experience levels.
```

Comment:
Sorry this read funny like the partners were for all ages. Can we change to this?

`The Hour of Code website has over 100 computer science activities for all ages and experience levels, developed by Code.org and partners.`